.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
This file must not be excluded. It must be preserved in the git tree as it will be used for deployment. Only .env.local file might be excluded.